achievements.py

```diff
@@ -1,5 +1,6 @@
 import random
 import pendulum
+import datetime
```

Comment:
As the dates in MongoDB are stored as python datetime object, I would stick to datetime instead of pendulum to avoid unnecessary conversions (or am I missing something?). Until now, you use pendulum just once in `achievements.py`, in line 37 which is given below:
```
    coffees = Consumption.objects(user=consumption.user, date__gte=pendulum.today)
```
This could be simply replaced by `datetime.date.today()`.

Reply:
Pendulum is just a fancy wrapper around the standard datetime (https://pendulum.eustace.io). It adds a few cool functions to calculate with dates but is completely compatible with datetime objects.